src/Pomelo.EntityFrameworkCore.MySql/Migrations/MySqlMigrationsSqlGenerator.cs

```diff
@@ -316,13 +317,25 @@ protected override void ColumnDefinition([CanBeNull] string schema, [NotNull] st
                 type = TypeMapper.FindMapping(property).StoreType;
             }
 
-            var generatedOnAddAnnotation = annotatable[MySqlAnnotationNames.Prefix + MySqlAnnotationNames.ValueGeneratedOnAdd];
+	        var matchType = type;
+	        var matchLen = "";
```

Comment:
Would have to convert it from string to int.  This is in the migration builder, not on any runtime code.  I don't think there's any issue of SQL Injection as this only takes input from EF Annotations and Fluent API during migrations, right?